src/dyncfg-launchdarkly/src/lib.rs

```diff
@@ -201,6 +201,12 @@ impl<F: Fn(&ConfigUpdates, &ConfigSet) + Send> SyncedConfigSet<F> {
                     entry.name()
                 ),
             };
+            tracing::debug!(
+                "updating config value {} from {:?} to {:?}",
+                &entry.name(),
+                &entry.val(),
+                update
+            );
```

Comment:
I don't think we have sensitive values stored directly in dyncfg so this should be ok.